Gopkg.toml

```diff
@@ -68,8 +68,8 @@ required = [
 
 [[constraint]]
   name = "github.com/openshift/hive"
-  revision = "1c37153bc9a1aae16dc1a40a90328a2e2a13d1ab"
-  
+  branch = "v1"
```

Comment:
Recommend using a specific commit or branch master. Branch v1 should work but has been merged into master and stop receiving updates as far as I know. Branch v1 could disappear but a commit hash or master won't. Not a critical issue.

Reply:
:+1: Thanks Carl, I set this to the specific commit.